Tools/autotest/common.py

```diff
@@ -2139,6 +2176,7 @@ def test_dataflash_over_mavlink(self):
     def test_arm_feature(self):
         """Common feature to test."""
         # TEST ARMING/DISARM
+        self.set_parameter("ARMING_CHECK", 1)  # ensure that arming check are on
```

Comment:
I'd rather see an assert here that it is `1`!

Reply:
done